news/_posts/2019-01-30-slick-3.3.0-released.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: news
+title: Slick 3.3.0
+author: Renato Cavalcanti
+---
+We have just released Slick 3.3.0
```

Comment:
I published exactly the same text here:

https://github.com/slick/slick/releases/tag/v3.3.0